include/zephyr/posix/arpa/inet.h

```diff
@@ -7,11 +7,14 @@
 #define ZEPHYR_INCLUDE_POSIX_ARPA_INET_H_
 
 #include <zephyr/net/socket.h>
+#include <zephyr/posix/arpa/inet.h>
```

Comment:
does this file name refer to itself? Or is there another zephyr/posix/arpa/inet.h ?

Reply:
Nope - that's it. Good catch 😅

Reply:
Fixed!